### DISCUSSION TITLE
Is it possible to animate streamed chunks in ReactMarkdown?

### DISCUSSION THREAD

**Discussion by matined** [remarkjs/remark#1372] [Q&A]
I'm streaming LLM response into the `ReactMarkdown` component. Let's say I have `<ReactMarkdown>{streamedText}</ReactMarkdown>`. Do you know a way to apply some animations to the newly appearing chunks of a streamedText? I want the streaming look softer.

Thanks in advance!

**Comment by ChristianMurphy** [comment]
Welcome @matined 👋 

> Is it possible to animate streamed chunks in ReactMarkdown?

yes it is, through CSS.
See the last time this was asked for specifics on a starting point https://github.com/orgs/remarkjs/discussions/1321#discussioncomment-9243116

Or if you are looking for something premade, consider https://llm-ui.com/ which builds off the same libraries as `react-markdown`.
